From 40d696df809d47c1873e01a7cfa8e8a107c857d3 Mon Sep 17 00:00:00 2001
From: Christian Ege <k4230r6@gmail.com>
Date: Mon, 4 May 2015 14:05:10 +0200
Subject: [PATCH] Loading bootlogo with ext4load instead of movi

Maybe the movi command could have been used as-well. But the ext4load
was more handy to use.

Signed-off-by: Christian Ege <k4230r6@gmail.com>
---
 include/configs/hardkernel/odroidc.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/configs/hardkernel/odroidc.h b/include/configs/hardkernel/odroidc.h
index 2099f3b..03fd60d 100644
--- a/include/configs/hardkernel/odroidc.h
+++ b/include/configs/hardkernel/odroidc.h
@@ -158,10 +158,10 @@
         "initrd_high=60000000\0" \
         "loadaddr=" MK_STR(CONFIG_SYS_LOAD_ADDR) "\0" \
         "loadaddr_logo=0x14000000\0" \
-        "preloadlogo=logo size ${outputmode}; video open;" \
-                "video clear; video dev open ${outputmode};" \
-                "movi read logo 0 ${loadaddr_logo};" \
-                "bmp display ${loadaddr_logo}; bmp scale\0" \
+        "preloadlogo=logo size ${outputmode};video open;" \
+                "video clear;video dev open ${outputmode};" \
+                "ext4load mmc 0:1 ${loadaddr_logo} /boot/bootlogo.bmp;" \
+                "bmp display ${loadaddr_logo};bmp scale\0" \
         "vdac_config=0x10\0" \
         "video_dev=tvout\0"

--
2.1.4

